packages/slice-machine/lib/models/common/CustomType/fields.ts

```diff
@@ -7,11 +7,12 @@ export enum FieldType {
   Color = 'Color',
   Group = 'Group',
   StructuredText = 'StructuredText',
-  ContentRelationship = 'Link',
   Image = 'Image',
   Text = 'Text',
   Date = 'Date',
   Link = 'Link',
+  LinkToMedia = 'LinkToMedia',
```

Comment:
I think this one should be Link as well, and when it goes in the function that find the widget by type or config, it will find the right widget thanks to the "select" attribute